src/test/java/seedu/anilist/logic/commands/EditCommandTest.java

```diff
@@ -81,12 +81,12 @@ public void execute_noFieldSpecifiedUnfilteredList_success() {
 
     @Test
     public void execute_filteredList_success() {
-        showPersonAtIndex(model, INDEX_FIRST_ANIME);
+        showAnimeAtIndex(model, INDEX_FIRST_ANIME);
 
         Anime animeInFilteredList = model.getFilteredAnimeList().get(INDEX_FIRST_ANIME.getZeroBased());
-        Anime editedAnime = new AnimeBuilder(animeInFilteredList).withName(VALID_NAME_BOB).build();
+        Anime editedAnime = new AnimeBuilder(animeInFilteredList).withName(VALID_NAME_BNHA).build();
         EditCommand editCommand = new EditCommand(INDEX_FIRST_ANIME,
-                new EditPersonDescriptorBuilder().withName(VALID_NAME_BOB).build());
+                new EditAnimeDescriptorBuilder().withName(VALID_NAME_BNHA).build());
 
         String expectedMessage = String.format(EditCommand.MESSAGE_EDIT_PERSON_SUCCESS, editedAnime);
```

Comment:
Is this part cannot change cos its part of edit command?

Reply:
Ok I will refactor edit command as well, wasn't sure previously if I should do that in this pr